src/cli/multi.rs

```diff
@@ -26,6 +26,9 @@ pub struct CliMulti {
     /// The directory into which network units are written
     #[arg(long = "network-units", value_name = "path")]
     network_units_dir: Option<String>,
+    /// The directory into which netplan configs are written
+    #[arg(long = "netplan-configs", value_name = "path")]
```

Comment:
Since we only write down one config, should this be

```suggestion
    #[arg(long = "netplan-config", value_name = "path")]
```

?

Reply:
My idea was that the Equinix Metal provider could also offer Netplan instead of networkd directly but yes, since it works to write one large config file instead of many being merged, it would still be ok in the singular even then.